edi_835_parser/segments/financial_information.py

```diff
@@ -28,12 +33,13 @@ def __init__(self, segment: str):
 		self.id = get_element(segment, 7)
 		self.acc_qualifier = get_element(segment, 8)
 		self.sender_acc_no = get_element(segment, 9)
-		self.routing_number = get_element(segment, 13)
-		self.transaction_date = get_element(segment, 16)
 		self.payer_id = get_element(segment, 10)
+		self.origin_company_code = get_element(segment, 11)
+		self.dfi_id_number_qualifier = get_element(segment, 12)
+		self.routing_number = get_element(segment, 13)
 		self.account_no_qualifier = get_element(segment, 14)
 		self.receiver_or_provider_acc_no = get_element(segment, 15)
-		self.origin_company_code = get_element(segment, 11)
+		self.transaction_date = get_element(segment, 16)
```

Comment:
This is just sorting the segments to be in order.

Reply:
👍